.github/actions/verify-token/src/utils.ts

```diff
@@ -0,0 +1,18 @@
+import path from "path";
+import * as process from "process";
+
+export function getEnv(name: string): string {
+  const res = process.env[name];
+  if (typeof res === "undefined") {
+    throw new Error(`missing env: ${name}`);
```

Comment:
do we also need to verify that res is not the empty string?

Reply:
done